### PR TITLE
Feature/footer

### PR DESCRIPTION
**Pull request by yohdev-travis**

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -1,42 +1,99 @@
...
-              Proudly presented by Henrik Wirth
-            </a>
-          </p>
-        </div>
+    <section className="footer">
+      <Container>
+        <Row className="justify-content-md-between">
+          <Col md={2} sm={12}>
+            <div className="footer-block one">
+              <img src={logo} alt="Logo" className="img-fluid" />
```
Comment (yohdev-von) [inline src/components/Footer.js]: This needs to be changed from HTML img tag to Gatsby Image.

File: src/components/Footer.js [hunk 2]
```diff
@@ -1,42 +1,159 @@
...
+                  <Link to="/">Our Process</Link>
+                  <Link to="/">FAQ's</Link>
+                </div>
+              </div>
+            </Col>
+            <Col md={3} sm={12}>
+              <div className="footer-block five">
+                <a href="/" className="btn gradient-btn">
+                  Contact Us
+                </a>
```
Comment (yohdev-von) [inline src/components/Footer.js]: This needs to be Bootstrap Button 

I have made a similar comment on the Nav task. These both need to match as Bootstrap Buttons.
[Nav](https://github.com/yohdev/gatsby-wp/pull/8#discussion_r893819584)

File: src/components/Footer.js [hunk 3]
```diff
@@ -1,42 +1,159 @@
...
+                    <FontAwesomeIcon icon={faListCheck} />
+                  </div>
+                  <h3 className="heading">Needs Assessment</h3>
+                  <p className="text">
+                    Let us know what you need, and we'll find the best solution
+                    to satisfy your business goals.
+                  </p>
+                  <button className="btn gradient-btn">
+                    Fill The Assessment Out
+                  </button>
```
Comment (yohdev-von) [inline src/components/Footer.js]: Needs to be Bootstrap button.

File: src/components/Footer.js [hunk 4]
```diff
@@ -1,42 +1,159 @@
...
```
Comment (yohdev-von) [inline src/components/Footer.js]: Needs to be Bootstrap button.

**Review by yohdev-von — CHANGES_REQUESTED** [review]

File: src/components/Footer.js (hunk 4 above)
Comment (yohdev-travis) [reply src/components/Footer.js]: Image has been updated to Gatsby Static Image and buttons are now React Bootstrap buttons.

**Review by yohdev-von — APPROVED** [review]

**Pull request merged**